src/distrib/make_installer.sh

```diff
@@ -32,7 +26,18 @@ case \$targetarch in
   *armv*) targetarch="armhf" ;;
 esac
 
-archive=ocaml-platform-$VERSION-\$static_targetos-\$targetarch.tar
+targetos=\$(uname -s | tr '[:upper:]' '[:lower:]' || true)
+case \$targetos in
+  "darwin")
+    static_targetos=macos
+    targetos=macos
```

Comment:
Redefining this variable doesn't seem to be a good idea if you expect it to be the result of `uname`. But it doesn't seem to be used later in the script, we can probably delete this line?

```suggestion
    static_targetos=macos
```